frontend/src/pages/zaaktypen/Zaaktypen.tsx

```diff
@@ -57,6 +58,10 @@ export function ZaaktypenPage() {
           href="zaaktypen/create"
           key="create-zaaktype"
           variant="primary"
+          onClick={(e) => {
+            e.preventDefault();
+            return navigate((e.target as HTMLAnchorElement).pathname);
+          }}
```

Comment:
Maybe also use the NavLink like in the InformatieObjectType page, so that it's consisten?